### ISSUE TITLE
MSUnmerged: Manual cleanup of /store/unmerged area at T2_CH_CERN

### ISSUE THREAD

**Issue by todor-ivanov** [dmwm/WMCore#11904]
**Impact of the bug**
T2_CH_CERN site 

**Describe the bug**
As a consequence of the bug: https://github.com/dmwm/WMCore/issues/11893 we have accumulated almost  1 PB of data  in the `/store/unmerged` area at `T2_CH_CERN` . The combination of the previously prolonged fix of the Gfal recursive errors  leads to a lot of empty directories being accumulating as well. This two govern the necessity of initial manual intervention for cleaning those either trough 
* a standalone run of the MSUnmerged at T2_CH_CERN
* or by manually implementing the logic of filtering out the protected  LFNs and doing the deletions directly on an `/eos` mount point. 

For the later we have to either ask the VOC for delete permissions at `/eos/cms/store/unmerged` area or provide a list of deletable objects and let him or DM Team to do the deletions  

**How to reproduce it**
N/A

**Expected behavior**
Follow  up on the manual intervention and delete all unneeded objects 

**Additional context and error message**
None

**Comment by todor-ivanov** [comment]
And Here is the Jira ticket for requesting access to the `/store/unmerged` area at CERN for doing this cleanup process: https://its.cern.ch/jira/projects/CMSVOC/issues/CMSVOC-530?filter=allopenissues

**Comment by amaltaro** [comment]
@todor-ivanov before getting write access to the unmerged area, why don't you temporarily increase the resource requirements of the service? Such that it can go through the first run and get back into a more normal load.

Please also help me understanding why we have accumulated so much unneeded data at CERN. Is it:
a) because we were **never** able to remove directories, hence leaving directories behind since day 1.
b) because at some point we were no longer able to remove directories due to some misconfiguration (change in the MSUnmerged certificate, perhaps?)
c) because it's been long since we managed to remove anything at T2_CH_CERN
d) because of the migration to webdav
e) or all of the above?

**Comment by todor-ivanov** [comment]
> For the watchdog kill, are you seeing it in kubernetes or in your own environment?

It is the our Watchdog who is killing the process with  `SIGSEGV` (sig 11).  Which is basically a sign of memory depletion.

I do not have the permissions  to reopen this ticket, I hope @arooshap  have: https://its.cern.ch/jira/browse/CMSVOC-491?focusedId=6244590&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6244590

**Comment by klannon** [comment]
I don't understand why this is a high priority issue.  This seems like it should be an "Operations" issue.  We didn't plan for this.  It's extra and only being considered because of it's operational impact.  @todor-ivanov, if this is correct can you please recategorize this as an Operations issue?

**Comment by todor-ivanov** [comment]
Hi @klannon I did categorize it and labeled it as Operations from the very beginning.

**Comment by klannon** [comment]
@todor-ivanov That is not correct.  You need to set `QPrio` to "Operations."  I have just done that for this issue.

**Comment by todor-ivanov** [comment]
**Finally** CERN is clean: [1] 
For more technical details on how the actual cleanup process went, with the code improvements I did, please read the information provided with the associated (but not merged) PR:  https://github.com/dmwm/WMCore/pull/11916 and this comment in particular: https://github.com/dmwm/WMCore/pull/11916#issuecomment-1973322541 

I am closing this issue now. 

[1] 
https://monit-grafana.cern.ch/goto/BrePj6oSg?orgId=11 

![EosCmsStoreUnmerged_2024-03-01_16-09-55](https://github.com/dmwm/WMCore/assets/8182807/30b00b89-827f-4480-9cf7-a170b726f5c4)

**Comment by amaltaro** [comment]
Awesome! Should we reinsert T2_CH_CERN into the MSUnmerged configuration - once the unmerged consistency monitor dump is in a more manageable size?

**Comment by todor-ivanov** [comment]
I think we can do it immediately
